torch/utils/_cxx_pytree.py

```diff
@@ -869,60 +874,6 @@ def tree_any_only(
     return any(pred(x) for x in flat_args if isinstance(x, type_or_types))
 
 
-def broadcast_prefix(
```

Comment:
This function is not publicly used and it does not exist in the Python pytree. The logic is moved to `_broadcast_to_and_flatten`.